tests/unit/bokeh/test_client_server.py

```diff
@@ -511,44 +507,42 @@ def client_lacks_root():
 
             assert dict(a=0, b=1, c=2, d=3, e=4) == result.values
 
-    @pytest.mark.asyncio
-    async def test_server_session_timeout_async(self, ManagedServerLoop):
-        application = Application()
-        with ManagedServerLoop(application) as server:
-            doc = document.Document()
-            doc.add_root(DictModel())
+    # def test_server_session_timeout_async(self, ManagedServerLoop):
```

Comment:
It would have been better to skip these tests, so that we have some indication in test output about their existence. xfailing would be even better, but that hangs tests. I can push a commit as I already did the changes locally.

Reply:
@mattpap OK Please do push as soon as you can.